PersianTools/Modules/LegalId.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Linq;
+
+namespace PersianTools.Modules
+{
+    public static class LegalId
+    {
+        public static bool Verify(string legalId)
+        {
+            const int legalLength = 11;
```

Comment:
According to this [link](https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/identifier-names),
Please 

> Use PascalCase for constant names, both fields and local constants.